packages/lib/commands/synchronize.ts

```diff
@@ -35,16 +35,20 @@ export const runtime = (): CommandRuntime => {
 					return 'auth';
 				}
 
-				reg.logger().info('Not authentified with sync target - please check your credential.');
+				reg.logger().error('Not authenticated with sync target - please check your credentials.');
 				return 'error';
 			}
 
 			let sync = null;
 			try {
 				sync = await reg.syncTarget().synchronizer();
 			} catch (error) {
-				reg.logger().info('Could not acquire synchroniser:');
-				reg.logger().info(error);
+				reg.logger().error('Could not acquire synchroniser:');
+				reg.logger().error(error);
+				utils.store.dispatch({
+					type: 'SYNC_REPORT_UPDATE',
+					report: { errors: [error] },
```

Comment:
As it's going to affect all sync targets, and some might output not so useful error messages, please could you add more info? Just `error.message = "Could not initialise synchroniser: " + error.message` should do.

And that should be done in side-menu-shared too.

Reply:
"Could not initialise synchroniser" please. We use British English by default and "acquire synchronizer" doesn't mean much I think.

Reply:
@laurent22 Thank you for the comment. All the logs are now in British English.